.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,3 +1,12 @@
+---
+name: Bug Report
+about: Create an issue that currently exist with JustAskDavidB
+title: ''
+labels: ''
```

Comment:
Add 'bug' as one of the labels because bug reports will always have the 'bug' label.

Reply:
This can be done by entering into a new line under "labels:", adding a dash, and typing the label that is to be automatically assigned. Like this:
![image](https://github.com/user-attachments/assets/64524350-d6fd-429e-a40e-ab0cb94679f9)

Reply:
@andrew-zhouu  this has been resolved. thanks.